client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js

```diff
@@ -132,4 +135,27 @@ export function areOutputParametersSupported(element) {
   ]);
 }
 
+export function createElement(type, parent, factory, properties) {
+  return elementHelper.createElement(type, properties, parent, factory);
+}
+
+export function createIOMapping(parent, bpmnFactory, properties) {
+  return createElement('zeebe:IoMapping', parent, bpmnFactory, properties);
+}
 
+/**
+ * Get getter function for IOMapping parameters according to provided property name
+ *
+ * @param {string} property
+ *
+ * @returns {Function} Getter function for the IOMapping parameters according to provided property name
+ */
+export function determineParamGetFunc(property) {
+  if (property == 'inputParameters') {
```

Comment:
I'd rather not use `==` comparison anywhere.

```suggestion
  if (property === 'inputParameters') {
```

Reply:
Done